src/main.h

```diff
@@ -2,13 +2,14 @@
 #define MAIN_H
 
 #include "types.h"
-// 20836
-// 655
-#define GFX_POOL_SIZE 20836
+
+#define MTX_POOL_SIZE 0x0683
+#define GFX_POOL_SIZE 0x1D4C
 struct GfxPool {
-    Gfx buffer[GFX_POOL_SIZE];
-    struct SPTask spTask;
-};
+    /* 0x00000 */ Mtx matrixPool[MTX_POOL_SIZE];
+    /* 0x1A0C0 */ Gfx gfxPool[GFX_POOL_SIZE];
```

Comment:
SM64's struct is laid out as such:
```
struct GfxPool {
    Gfx buffer[GFX_POOL_SIZE];
    struct SPTask spTask;
};
```
Thoughts on `gfxPool` vs just `buffer`? Probably don't need to include the struct name in the var name.

Reply:
I felt it would help with readability if the member name had some sort of indicator of what type it is. The member being named the same as the struct type is a little weird, but I don't find it terribly confusing. I'm open to suggestions though

Reply:
I think buffer better describes what it is. And when we see this being used you generally see: `gfxPool->buffer` anyway
ex: `&gGfxPool->buffer[32])` vs ` gGfxPool->gfxPool;`

Reply:
Not sure what the name the mtx tho

Reply:
Okay, what you have is fine. But rename ` matrixPool[MTX_POOL_SIZE];` to `mtxPool[]`

Reply:
lol, just reverted gfxPool back to buffer. I'll update mtxPool, should I leave gfxPool as buffer or should I undo the revert?